<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touring Simulator</string>
  </property>
  <property name="whatsThis">
   <string>A Touring Simulation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="LoginPage">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="3">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>256</number>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="LogInButton">
            <property name="text">
             <string>登陆</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SignUpButton">
            <property name="text">
             <string>注册</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>328</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>328</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>328</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>328</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>236</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>237</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="welcom_label">
         <property name="text">
          <string>欢迎使用</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MainPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="OrderPageButton">
         <property name="text">
          <string>预定行程</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="QueryPathPageButton">
         <property name="text">
          <string>路线查询</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="StatePageButton">
         <property name="text">
          <string>状态查询</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="SimulationPageButton">
         <property name="text">
          <string>模拟旅行</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="ExitButton">
         <property name="text">
          <string>退出程序</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="StatePage">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QPushButton" name="StatePageToMenuButton">
         <property name="text">
          <string>返回主菜单</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextBrowser" name="StatetextBrowser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="QueryPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,5,1">
           <property name="spacing">
            <number>50</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalOriginLayout" stretch="1,5">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="origin_label">
               <property name="text">
                <string>始发地</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="origin_comboBox">
               <item>
                <property name="text">
                 <string>济南</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>石家庄</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>长春</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>哈尔滨</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>沈阳</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>呼和浩特</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>乌鲁木齐</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>兰州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>银川</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>太原</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>西安</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>郑州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>合肥</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>南京</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>杭州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>福州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>广州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>南昌</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>海口</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>南宁</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>贵阳</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>长沙</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>武汉</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>成都</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>昆明</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>西宁</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>天津</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>上海</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>重庆</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>北京</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>拉萨</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalTargetLayout" stretch="1,5">
             <item>
              <widget class="QLabel" name="target_label">
               <property name="text">
                <string>目的地</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="target_comboBox">
               <item>
                <property name="text">
                 <string>济南</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>石家庄</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>长春</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>哈尔滨</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>沈阳</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>呼和浩特</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>乌鲁木齐</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>兰州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>银川</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>太原</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>西安</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>郑州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>合肥</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>南京</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>杭州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>福州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>广州</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>南昌</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>海口</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>南宁</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>贵阳</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>长沙</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>武汉</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>成都</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>昆明</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>西宁</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>天津</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>上海</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>重庆</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>北京</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>拉萨</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="QueryPathButton">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="Path_tableWidget">
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>27</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>序号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>始发地</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>目的地</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>交通方式</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>出发时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>到达时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>价格</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="QueryPageToMenuButton">
         <property name="text">
          <string>返回主菜单</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="OrderPage">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>577</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_18">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>880</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="OrderPageToMenuButton">
         <property name="text">
          <string>返回主菜单</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>released()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>859</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
